Assets/ModScripts/CruelModkitScript.cs

```diff
@@ -499,24 +496,40 @@ IEnumerator PlaySolveAnim()
     // Animations but also sets up Puzzle class
     void AssignHandlers()
     {
-        //SelectModule = "Timer Timings";
         switch (SelectModule)
         {
-            case "Metered Button":
-                TargetComponents = 2;
-                Puzzle = new MeteredButton(this, ModuleID, Info, true, TargetComponents);
-                break;
             case "Timer Timings":
                 TargetComponents = 0;
-                Puzzle = new TimerTimings(this, ModuleID, Info, true, TargetComponents);
+                Puzzle = new TimerTimings(this, ModuleID, Info, TargetComponents);
                 break;
-            case "Test Puzzle":
-                TargetComponents = 255;
-                Puzzle = new TestPuzzle(this, ModuleID, Info, true, TargetComponents);
+            case "Unscrew Maze":
+                TargetComponents = 3;
```

Comment:
For all TargetComponents in this switch case : use bitwise operations with the flag enum instead of a number for way better readability
```suggestion
                TargetComponents = (byte)(ComponentsEnum.Bulbs & ComponentsEnum.Arrows);
```